log-insights/setup/crunchy-bridge/01_create_helper_function_and_test.mdx

```diff
@@ -17,6 +17,10 @@ Log output for Crunchy Bridge database servers is collected directly from the da
 
 If you are using the application user, an additional function needs to be created to grant access to the logs (**run the following with superuser credentials**):
 
+```sql
+GRANT pg_monitor TO application;
+```
```

Comment:
This will end up in a separate code block, right? That might look a little weird. Maybe we should be more explicit:

````suggestion
If you are using the application user, first, **connect with superuser credentials** and ensure this user has the `pg_monitor` role:
```sql
GRANT pg_monitor TO application;
```
Then, create a function to grant this user access to the logs:
````

Reply:
Ah yeah, I got lazy, looking better 👍